dbfit-java/core/src/test/java/dbfit/util/DbParameterAccessorsMapBuilderTest.java

```diff
@@ -7,10 +8,12 @@
 import static org.hamcrest.Matchers.*;
 
 import java.util.Map;
+import java.util.HashMap;
```

Comment:
I checked locally - I still do not see any `Map` nor `HashMap` in `DbParameterAccessorsMapBuilderTest.java`...